src/components/AccountsList/index.js

```diff
@@ -24,30 +26,53 @@ function AccountsList({
   const handleClose = () => setOpen(false);
   const [listAccounts, setListAccounts] = useState(accounts);
   const [newAccounts, setNewAccounts] = useState("");
-  const fetcher = (url) => fetchJson(url);
-  const { data, mutate } = useSWR(apiUrl, fetcher);
+  const [existingAccounts, setExistingAccounts] = useState(accounts);
+  const [page, setPage] = useState();
+  const [pageSize, setPageSize] = useState(3);
+  const fetcher = (url, pg, pSize) => {
+    const queryString = getQueryString({
+      page: pg,
+      pageSize: pSize,
+    });
+    let listURL = url;
+    if (queryString) {
+      listURL = `${listURL}&&${queryString}`;
+    }
+    return fetchJson(listURL);
+  };
+
+  const { data, mutate } = useSWR(
+    [`${apiUrl}/?accounts=true`, page, pageSize],
+    fetcher
+  );
+
+  const { data: accountsData, mutate: mutateAccountsData } = useSWR(
+    `${apiUrl}/?accounts=true`,
+    fetcher
+  );
```

Comment:
@gertie-sheshe I didn't know you call `useSWR` hook twice. 

Anyway, isn't this one and the same?  Both using the same url `${apiUrl}/?accounts=true`

Reply:
Yes you can. Did it because we need all accounts during addition of a new one. @KhadijaMahanga 

No they aren't the same, the first one has the url plus arguments to pass to the fetcher function i.e [`${apiUrl}/?accounts=true`, page, pageSize]

Reply:
@gertie-sheshe if we are using it during creation, then I don't think it should be in a hook. It should be called during creation only

Reply:
Fixed